### ISSUE TITLE
Replace offlineimap with mbsync for importer

### ISSUE THREAD

**Issue by famz** [patchew-project/patchew#59]
In my recent experience, I found mbsync much more efficient than offlineimap. It may not matter much for long-running importers since at every iteration one should only see a handful of new messages, but on higher traffic, or when `imap_delete_after_import=n`, or when the importer has been paused for various reasons for a few days, this can make a difference.

**Comment by famz** [comment]
Nope, but I believe it should be fairly straightforward. Is this urgent?

**Comment by bonzini** [comment]
No, I was just wondering what the state was, because I ran into some kind of bug today with the importer after it was stopped and restarted. :)
